packages/shared/src/features/onboarding/shared/FunnelStepCtaWrapper.tsx

```diff
@@ -0,0 +1,25 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+import type { ButtonProps } from '../../../components/buttons/Button';
+import { Button, ButtonVariant } from '../../../components/buttons/Button';
+
+export function FunnelStepCtaWrapper({
+  children,
+  ...props
+}: ButtonProps<'button'>): ReactElement {
+  return (
+    <div className="relative flex flex-col gap-4">
+      {children}
+      <div className="sticky bottom-2">
+        <Button
+          type="button"
+          className="w-full"
+          variant={ButtonVariant.Primary}
+          {...props}
+        >
+          Next
```

Comment:
Should this be dynamic?

Reply:
Not required now, but added to be future-proof since is a simple change!